Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -107,14 +77,24 @@ internal T Execute<T>(Func<TransactionExecutor, T> func)
                 transaction.Commit();
                 return returnedValue;
             }
+            catch (QldbTransactionException qte)
+            {
+                throw qte;
+            }
             catch (InvalidSessionException ise)
             {
-                this.isAlive = false;
-                throw new RetriableException(transactionId, false, ise);
+                if (Regex.Match(ise.Message, @"Transaction\s.*\shas\sexpired").Success)
```

Comment:
This is extremely brittle and should not be done, what if the error message is not English?

Reply:
We should raise this again to ensure it's tracked on the service-side.

Reply:
Ok - as long as there's a ticket tracking this.